### DISCUSSION TITLE
Make Personal Lists Easier/More Convenient to Find

### DISCUSSION THREAD

**Discussion by jenlith** [community/community#19034] [Other Feature Feedback, Questions, & Ideas]
I know lists are under "Stars," but that wasn't obvious. They just happened to be there. (I've starred a couple repos along with adding them to a list.) I'd suggest at least renaming that section to "Stars & Lists"

**Comment by manuletsgo** [comment]
yeah, I fully agree. Maybe create a separate section side by Stars?

**Comment by Terry316** [comment]
I Agree, it sometimes becomes challenging to know the next steps.

**Comment by Mehul2203** [comment]
Yes, I partially agree with you @jenlith. I have also starred a couple of repositories.

**Comment by Tremeschin** [comment]
In my opinion, lists makes sense to be located after the pinned repositories as a collapsing list on one's profile.

This way we build a story "Hey, this is me", "Here's my best work", "And here's stuff I find (most) interesting / am working on (nitpicked list)".

This way we keep the "Stars" tab as something more of an "here's (all) the repos I support / find interesting"

Anyways, that's my two cents on the topic.
